scripts/goldpolish-target-extract-seq.py

```diff
@@ -41,52 +42,53 @@ def parse_args():
 def make_coord_dict(bed):
     """Creates a dictionary of tuples representing regions to polish"""
     coord_dict_2 = {}
+    Coordinate = namedtuple("Coordinate", "start end")
```

Comment:
To avoid re-defining it in different functions and making the scope potentially an issue, just put this definition in the global scope (like you would for a class)